Sources/OTelConventions/Generated/OTelAttribute+http.swift

```diff
@@ -0,0 +1,113 @@
+            /// 
+            /// - Examples:
+            ///     - `GET`
+            ///     - `POST`
+            ///     - `HEAD`
+            public static let method = "http.request.method"
+        
+            /// `http.request.method_original`: Original HTTP method sent by the client in the request line.
+            /// 
+            /// - Stability: stable
+            /// 
+            /// - Type: string
+            /// 
+            /// - Examples:
+            ///     - `GeT`
+            ///     - `ACL`
+            ///     - `foo`
+            public static let methodOriginal = "http.request.method_original"
+        
+            /// `http.request.resend_count`: The ordinal number of request resending attempt (for any reason, including redirects). 
+            /// 
+            /// - Stability: stable
+            /// 
+            /// - Type: int
+            /// 
+            /// The resend count SHOULD be updated each time an HTTP request gets resent by the client, regardless of what was the cause of the resending (e.g. redirection, authorization failure, 503 Server Unavailable, network issues, or any other). 
+            /// 
+            /// - Example: `3`
+            public static let resendCount = "http.request.resend_count"
+        
+        
+        }
+    
+        /// `http.response` namespace
+        public enum response {
+            /// `http.response.header`: HTTP response headers, `<key>` being the normalized HTTP Header name (lowercase), the value being the header values. 
+            /// 
+            /// - Stability: stable
+            /// 
+            /// - Type: templateStringArray
+            /// 
+            /// Instrumentations SHOULD require an explicit configuration of which headers are to be captured. Including all response headers can be a security risk - explicit configuration helps avoid leaking sensitive information. Users MAY explicitly configure instrumentations to capture them even though it is not recommended. The attribute value MUST consist of either multiple header values as an array of strings or a single-item array containing a possibly comma-concatenated string, depending on the way the HTTP library provides access to headers. 
+            /// 
+            /// - Examples:
+            ///     - `http.response.header.content-type=["application/json"]`
+            ///     - `http.response.header.my-custom-header=["abc", "def"]`
+            public static let header = "http.response.header"
+        
+            /// `http.response.status_code`: [HTTP response status code](https://tools.ietf.org/html/rfc7231#section-6).
+            /// 
+            /// - Stability: stable
+            /// 
+            /// - Type: int
+            /// 
+            /// - Example: `200`
+            public static let statusCode = "http.response.status_code"
+        
+        
+        }
+    }
+}
```

Comment:
Speaking of whitespace, do we want to add final newlines via the generator, if only to please GitHub?

Reply:
Sure, I've added this in https://github.com/swift-otel/swift-otel-semantic-conventions/pull/2/commits/229fcbc96e0f694e2c6c4b99155a84a7fc0fb8d4